core/src/main/java/org/testcontainers/dockerclient/RootlessDockerClientProviderStrategy.java

```diff
@@ -0,0 +1,61 @@
+package org.testcontainers.dockerclient;
+
+import org.apache.commons.lang.SystemUtils;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ *
+ * @deprecated this class is used by the SPI and should not be used directly
+ */
+@Deprecated
+public final class RootlessDockerClientProviderStrategy extends DockerClientProviderStrategy {
+    protected static final String DOCKER_SOCK_PATH = "/run/user/1000/docker.sock";
```

Comment:
Slightly pedantic point, but I'm assuming we'll have to make the `1000` reflect the current user's uid. I guess this may become more necessary once testing rootless with cloud CI.